src/applications/static-pages/health-care-manage-benefits/get-medical-records-page/components/AuthContent/index.js

```diff
@@ -20,303 +17,242 @@ const AuthContent = ({
+            to take responsibility for protecting that information.
+          </p>
+        </va-accordion-item>
+        <va-accordion-item
+          header="How does VA share my health information with providers outside VA?"
+          id="fourth"
+        >
+          <p>
+            The Veterans Health Information Exchange (VHIE) program lets us
+            securely share your health information with participating community
+            care providers in our network. VHIE also lets us share your
+            information with the Department of Defense.
+          </p>
+          <p>
+            VHIE gives your health care providers a more complete understanding
+            of your health record. This more complete understanding can help
+            your providers make more informed treatment decisions. We share your
+            health information with participating community providers only when
+            they’re treating you.
+          </p>
+          <p>
+            If you don’t want us to share your information through VHIE, you can
+            opt out at any time.
+          </p>
+          <a href="/resources/the-veterans-health-information-exchange-vhie">
+            Learn more about VHIE
+          </a>
+        </va-accordion-item>
+        <va-accordion-item header="What if I have more questions?" id="fifth">
+          <h4>For My HealtheVet questions</h4>
+          <p>You can get more information in any of these ways:</p>
+          <ul>
+            <li>
+              Read the FAQs pages on the My HealtheVet web portal
+              <br />
+              <a href="https://www.myhealth.va.gov/mhv-portal-web/web/myhealthevet/faqs-blue-button">
+                VA Blue Button FAQs
+              </a>
+              <br />
+              <a href="https://www.myhealth.va.gov/mhv-portal-web/web/myhealthevet/faqs-health-summary">
+                VA health summary FAQs
+              </a>
+              <br />
+              <a href="https://www.myhealth.va.gov/mhv-portal-web/web/myhealthevet/what-are-va-medical-images-and-reports-faqs">
+                VA medical images and reports FAQs
+              </a>
+            </li>
+            <li>
+              Call the My HealtheVet help desk at{' '}
+              <a href="tel:18773270022" aria-label="8 7 7. 3 2 7. 0 0 2 2.">
+                877-327-0022
+              </a>{' '}
+              (<va-telephone contact={CONTACTS.HELP_TTY} tty />
+              ). We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+            </li>
+            <li>
+              <a href="https://www.myhealth.va.gov/contact-us">
+                Contact us online through My HealtheVet
+              </a>
+            </li>
+          </ul>
+          <h4>For My VA Health questions</h4>
+          <p>
+            Call My VA Health support anytime at{' '}
+            <a href="tel:18773270022" aria-label="8 0 0. 9 6 2. 1 0 2 4.">
+              800-962-1024
+            </a>
```

Comment:
Same here. One advantage of using `<va-telephone>` with the `contact` prop is that there's no chance of mismatching numbers. This `tel` link has an 877 number but the actual text and the `aria-label` are showing an 800 number.